application/mongo_client.py

```diff
@@ -16,10 +17,67 @@ def __init__(self):
         self.client = pymongo.MongoClient(f"mongodb+srv://{self.MONGO_USERNAME}:{self.MONGO_PASSWORD}@{self.MONGO_HOST}/?retryWrites=true&w=majority")
         self.db = self.client[self.MONGO_TODO_APP_DATABASE]
 
+    # User management.
+
+    @property
+    def users(self):
+        users = []
+        raw_users = self.db['users'].find()
+        for raw_user in raw_users:
+            user = User(raw_user['_id'], raw_user['role'])
+            users.append(user)
+        return users
+
+    def user_exists(self, github_id):
+        github_id = int(github_id)
+        return self.db['users'].count_documents({'_id': github_id}) > 0
+
+    def get_user(self, github_id):
+        github_id = int(github_id)
+        if self.user_exists(github_id):
+            raw_user = self.db['users'].find_one({'_id': github_id})
+            return User(raw_user['_id'], raw_user['role'])
+        else:
+            ## TODO: Work out a more sensible return type/functionality.
+            return None
+
+    def add_user(self, github_id, role = "reader"):
+        github_id = int(github_id)
+        if self.user_exists(github_id):
+            message = f"""User '{github_id}' already exists in database. Use the edit_user_role method if you want to change their role."""
```

Comment:
You have (and return) these messages in a couple of methods, but I can't see anywhere in your code that actually uses them, and it's not particularly clear that from the function name why this would return a string, so this would likely be cleaner without them